packages/cdktf/lib/tfExpression.ts

```diff
@@ -194,6 +194,11 @@ function markAsInner(arg: any) {
   }
 }
 
+export function insideTfExpression(arg: any) {
```

Comment:
nit: I'm a bit torn as I'm not quite happy with the name but don't know a better one 😅
Maybe add a docstring which describes something like "marks `arg` as being used in an inner Terraform expression and returns it for further use"?